sdks/python/apache_beam/dataframe/pandas_doctests_test.py

```diff
@@ -554,6 +554,8 @@ def test_string_tests(self):
                 's.str.repeat(repeats=[1, 2, 3])'
             ],
             f'{module_name}.str_repeat': ['s.str.repeat(repeats=[1, 2, 3])'],
+            # get_dummies examples are not casted to Categorical type
+            # Must be Categorical or Bool to work in Beam
             f'{module_name}.StringMethods.get_dummies': ['*'],
             f'{module_name}.str_get_dummies': ['*'],
```

Comment:
Should we keep the wildcard `*`, or change it to explicitly add the two expressions from the docs? Technically, we are now implementing `get_dummies`, but just for specific data types, so it may be misleading to say that all tests are still `wont_implement_ok`.

Reply:
I think it makes sense to keep the wildcard, I don't think pandas will ever have doctests for the specific case (CategoricalDtype) that we support.

Reply:
Sounds good.